MAUI/TextInputLayout/Migration.md

```diff
@@ -63,4 +63,6 @@ To migrate easier from Xamarin SfTextInputLayout to .NET MAUI SfTextInputLayout,
 </table>
 
 
+## Known Issues
 
+The Stroke Value isn't updated by manually input values due to an issue at the framework level in .NET 8.0. This problem arises because an implicit style isn't overridden by the manual value. However, you can resolve this by setting the Stroke Value using the Visual State Manager (VSM) through Style.
```

Comment:
Manually input values don't update the Stroke Value due to an issue at the framework level in .NET 8.0. This problem arises because the manual value doesn't override an implicit style. However, you can resolve this by setting the Stroke Value using the Visual State Manager (VSM) through Style.